app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -614,6 +662,17 @@ class GleanMetricsService(private val context: Context) : MetricsService {
 
             syncItems.set(syncedItems)
 
+            val toolbarPositionSelection =
+                if (context.settings().shouldUseFixedTopToolbar) {
+                    "fixed_top"
+                } else if (context.settings().shouldUseBottomToolbar) {
+                    "bottom"
+                } else {
+                    "top"
+                }
+
+            toolbarPosition.set(listOf(toolbarPositionSelection))
```

Comment:
Is this a new addition? If so, please document

Reply:
Nope! Part of the initial telemetry I forgot.